Labs/Lab Notebooks/05 - Azure Cognitive Search/embed_pdf_edits.ipynb

```diff
@@ -0,0 +1,399 @@
+      "Title: DTO Quick Guide.pdf\n",
+      "Score: 0.82576483\n",
+      "Content: Planning for and using DTO\n",
+      "\n",
+      "Employee Responsibilit\n",
+      "Category: hr docs\n",
+      "\n",
+      "{'category': 'hr docs', 'content': 'You may not use DTO, but you can apply any HHTO to cover unpaid time.\\xa0\\n\\nPlease contact Benefits@microsoft.com for more information on your specific situation.\\xa0\\xa0\\n\\nResources\\xa0\\n\\nDTO Policy\\xa0\\n\\nhttps://microsoft.sharepoint.com/sites/HRweb/SitePages/FAQ_DTO.aspx?xsdata=MDV8MDF8fGJjMmUzZjdlMGZjZjQwZGJjMDgzMDhkYmUwOTMzZjVmfD… 9/10\\n\\n\\xa0\\n\\x0c11/13/23, 1:05 PM\\n\\nFrequently asked questions (FAQ) about Discretionary Time Off (DTO)\\n\\nDTO Employee Quick Guide \\xa0\\nTime Away Guide \\xa0\\n\\n\\ue8e1 \\ue8e0 Did this content help you answer your question?\\n\\nhttps://microsoft.sharepoint.com/sites/HRweb/SitePages/FAQ_DTO.aspx?xsdata=MDV8MDF8fGJjMmUzZjdlMGZjZjQwZGJjMDgzMDhkYmUwOTMzZjVmf… 10/10', 'title': 'DTO FAQ.pdf', '@search.score': 0.8190258, '@search.reranker_score': None, '@search.highlights': None, '@search.captions': None}\n",
+      "Title: DTO FAQ.pdf\n",
+      "Score: 0.8190258\n",
+      "Content: You may not use DTO, but you can apply any HHTO to\n",
+      "Category: hr docs\n",
+      "\n",
+      "{'category': 'hr docs', 'content': 'https://microsoft.sharepoint.com/sites/HRweb/SitePages/DTOPolicy.aspx?xsdata=MDV8MDF8fGQ1MmE3ZTQwYjk4ZTQ3MzdiMmNhMDhkYmUwOTMyZTE…\\n\\n4/5\\n\\n\\xa0\\n\\xa0\\n\\xa0\\n\\xa0\\n\\xa0\\n\\xa0\\n\\x0c11/13/23, 1:06 PM\\n\\nDiscretionary Time Off (DTO) and Holidays Policy - US Salaried/Exempt Employees\\n\\n\\ue8e1 \\ue8e0 Did this content help you answer your question?\\n\\nhttps://microsoft.sharepoint.com/sites/HRweb/SitePages/DTOPolicy.aspx?xsdata=MDV8MDF8fGQ1MmE3ZTQwYjk4ZTQ3MzdiMmNhMDhkYmUwOTMyZTE…\\n\\n5/5', 'title': 'DTO Overview.pdf', '@search.score': 0.79536057, '@search.reranker_score': None, '@search.highlights': None, '@search.captions': None}\n",
```

Comment:
is there a way to automatically scrubs output before committing notebooks to prevent accidental check ins of data and to also reduce the line count difference?

Reply:
Ah yes, I should have cleared the outputs before pushing.. I'll fix that.